modules/reference/pages/systems/system-details/sd-virtualization.adoc

```diff
@@ -119,3 +121,176 @@ In the menu:System Details[Virtualization] tab of a traditionally registered bar
+
+[guimenu]``Port``::
+Port of the remote machine providing access to the pool.
+
+[guimenu]``iSCSI Qualified Name``::
+iSCSI Qualified Name of the iSCSI target.
+
+[guimenu]``IQN Initiator``::
+iSCSI Qualified Name of the initiator to connect to.
+
+[guimenu]``Username``::
+Username to use to connect to the remote storage
+
+[guimenu]``Passphrase``::
+Password to use to connect to the remote storage.
+For RBD pools, this is the base64 encoded key.
+
+[guimenu]``Source name``::
+Name of the storage pool source.
+
+[guimenu]``Directory``::
+Path to the directory of the pool.
+
+[guimenu]``Subdirectory``::
+Absolute path relative to the Gluster volume to use.
+
+[guimenu]``Adapter type``::
+Select the controller type, either``fc_host`` or ``scsi_host``.
+
+[guimenu]``Adapter name``::
+SCSI adapter name for ``scsi_host`` controller.
+
+[guimenu]``Adapter parent PCI address``::
+PCI address of the SCSI host in the ``0000:00:00.0`` format.
+Those can be listed using [systemitem]``lsscsi -v``.
+
+[guimenu]``Adapter parent address unique ID``::
+Unique ID of the SCSI host as found in ``/sys/class/scsi_host/host*/unique_id`` file.
+
+[guimenu]``Adapter parent name``::
+Name of the vport capable parent SCSI host of the virtual Host Bus Adapter (vHBA).
+
+[guimenu]``Adapter parent wwnn``::
+World Wide Node Name used by the ``fc_host`` to identify the vHBA parent device.
+
+[guimenu]``Adapter parent wwpn``::
+World Wide Port Name used by the ``fc_host`` to identify the vHBA parent device.
+
+[guimenu]``Adapter parent fabric wwn``::
+Fabric WWN of the vHBA parent device.
+
+[guimenu]``Adapter wwnn``::
+World Wide Node Name used by the ``fc_host`` to identify the vHBA device.
+
+[guimenu]``Adapter wwpn``::
+World Wide Port Name used by the ``fc_host`` to identify the vHBA device.
+
+[guimenu]``Manage vHBA deletion``::
+If checked the vHBA will be destroyed with the pool is destroyed.
+This property will be automatically activated if there is no existing vHBA.
+
+==== Target fields
+
+[guimenu]``Path``::
+Path to the storage pool mount or device on the virtual host.
+
+[guimenu]``Owner ID``::
+ID of the user owning the path folder or file.
+
+[guimenu]``Group ID``::
+ID of the group owning the path folder or file.
+
+[guimenu]``Permission mode``::
+Octal representation of the permissions to set on the path folder or file.
+
+[guimenu]``SELinux label``::
+SELinux label to set on the path folder or file.
+
+=== Editing a pool
+
+To edit the properties of storage pool, click the [guimenu]``Edit pool`` button in the pool's row.
+A dialog with the same fields than when creating a pool will be opened.
+
+=== Deleting a pool
+
+To delete a storage pool, click the [guimenu]``Delete`` button in the pool's row.
+By default the action only removes the storage pool definition but keeps the data.
+To really clear the data, click the checkbox in the opened modal dialog before click the [guimenu]``Delete`` confirmation button.
```

Comment:
```suggestion
To delete the pool data as well the storage pool definition, check the [guimenu]``something`` box before you click [guimenu]``Delete``.
```

Need to check the name of the box.

Reply:
Shouldn't it be `as well as the storage pool definition` ?

Reply:
The checkbox label is `Delete the pool, including the contained volumes. Cannot be undone`. Not sure if you want to have the complete label in the doc.